### ISSUE TITLE
Audio Call with screen turned off

### ISSUE THREAD

**Issue by horstle** [meshenger-app/meshenger-android#17]
Hi, I just tested this app a bit and though it worked great while the app was open, I was surprised that it would always end an audio call as soon as I'd turn off the screen go back to the homescreen.

I think it would be a great improvement if audio calls (and maybe even video calls) could continue to run with the screen turned off or in background.

**Comment by mwarning** [comment]
I agree, that would be nice. I assume you turned off the screen  manually. I have not found out how to keep audio running, yet.

**Comment by horstle** [comment]
I have no idea about Java, so I can't look into this myself, but maybe you could find out how they do it eg. in [NewPipe](https://github.com/TeamNewPipe/NewPipe).

**Comment by iddq** [comment]
https://github.com/meshenger-app/meshenger-android/issues/160
